libs/components/manifest/src/generator/generate-manifest.ts

```diff
@@ -47,6 +48,8 @@ export async function generateManifest(
   );
 
   const publicApi = await getPublicApi(projects);
+  await getDocumentation(publicApi, projects);
```

Comment:
This function returns `Promise<unknown>`. In a future pull request, this function will return a wholistic JSON object representing the entire public API's documentation.